src/js/common.js

```diff
@@ -280,6 +280,16 @@ function closeEntityModal(modelId) {
 }
 
 /**
+ * Acquire login URL from session and make transition
+ */
+function logoutManager() {
+    let ManagerInfo = JSON.parse(sessionStorage.ManagerInfo);
```

Comment:
Shouldn't it be localStorage?

Reply:
It is stored in localStorage, but after login it is re-stored from LocalStorage to SessionStorage (since SessionStorage is not shared before and after login)